Xamarin/MvvmSeed/MvvmSeed/MvvmSeed.Android/Linker/LinkerPleaseInclude.cs

```diff
@@ -0,0 +1,85 @@
+using System.Collections.Specialized;
+using System.Windows.Input;
+using Android.App;
+using Android.Views;
+using Android.Widget;
+
+namespace MvvmSeed.Android.Linker
+{
+    /// <summary>
+    /// This file is required to avoid android Linker wiping references to code not detected during static analysis.
+    /// (We could disable linking, but I rather not, no linking means x3 or x4 size for builds)
+    /// </summary>
+    public class LinkerPleaseInclude
```

Comment:
> What if we specified manually the bindings?

I'm not sure if I follow..

Right now bindings are defined on View's axml like this:

```
  <TextView
    android:layout_width="fill_parent"
    android:layout_height="wrap_content"
    local:MvxBind="Text SampleString"/>
```

You are proposing writing those bindings on View's code-behind? I'm not a big fan of that. The linker-stuff is ugly too, but it's not something that will require a lot of maintenance, it's not like we'll have to add code to that file often.. Most frameworks are aware of this and provides support for linker issues (e.g. [linker issue](https://github.com/MvvmCross/MvvmCross/issues/689) ).

I'm not saying no, I'm saying that writing bindings on code behind it's a heavy decision, if that's what you had in mind.

> Yes, we would end up with a huge file that has access to everything

That's pretty much what this file does 😄 . The linker is wiping references to `button.Click` for example, because there's not a single line on the code that references button's Click event. MvvmCross does use it internally through commands.. So that huge file will end up being just dummy reference to pieces of code that are not directly referenced by our own code, which is what this file is.

So, maybe I'm not understanding your proposal @AlphaGit